string.c

```diff
@@ -8158,10 +8158,11 @@ tr_find(unsigned int c, const char table[TR_TABLE_SIZE], VALUE del, VALUE nodel)
 
 /*
  *  call-seq:
- *     str.delete!([other_str]+)   -> str or nil
+ *    delete!(*selectors) -> new_string or nil
```

Comment:
I think this should also be fixed

```suggestion
 *    delete!(*selectors) -> self or nil
```